blog/2020-02-10-spring-boot-release-2.2.x.adoc

```diff
@@ -0,0 +1,55 @@
+---
+layout: blog
+title: Infinispan Spring Boot Starter released with Spring Boot 2.2.4.RELEASE
+date: '2020-02-10T12:00:00.000-00:00'
+author: Katia Aresti
+tags: [ "release", "spring boot", "spring" ]
+---
+
+Dear Infinispan and Spring Boot users,
+
+We have just released
+https://github.com/infinispan/infinispan-spring-boot[Infinispan Spring
+Boot] *2.1.8.Final* and *2.2.0.Final*.
+
+*2.1.8.Final* is using *Infinispan 9.4.17.Final and Spring Boot 2.2.2.RELEASE*
+
+*2.2.0.Final* is using *Infinispan 10.1.1.Final and Spring Boot 2.2.2.RELEASE* 
+
+
+== Important Marshalling Changes with Infinispan 10.x
+
+Infinispan in client/server mode has made important changes on https://infinispan.org/docs/stable/titles/developing/developing.html#marshalling[marshalling].
+Previous versions of Infinispan used JBoss Marshaller as a default.
+Now https://github.com/infinispan/protostream[ProtoStream] that works with https://developers.google.com/protocol-buffers[Protocol Buffers] is
+the default Marshaller.
+
+Unfortunately ProtoStream does not work yet with our Spring Cache and Session support.
+This might be supported in the future versions, but some important changes are required to do so in the actual implementation.
+
+Spring users in client/server mode, we need to configure two additional properties today:
+
+`infinispan.remote.marshaller=org.infinispan.commons.marshall.JavaSerializationMarshaller`
+`infinispan.remote.java-serial-whitelist=org.infinispan.tutorial.simple.spring.remote.*`
```

Comment:
why the simple tutorials? 
may I suggest use: 
`my.package.MyClass1,my.package.MyClass2,...`
?

Reply:
I copied the simple tutorials of infinispan sb classes